examples/kubeflow-jupyter-notebook/README.md

```diff
@@ -0,0 +1,66 @@
+# Using `Fairing` with Kubeflow's TfJob inside a Jupyter Notebook
+
+## Requirements
+
+### Kubeflow
+
+Kubeflow needs to be pre-installed in the cluster.
+
+### Knative Build Component
```

Comment:
I think that Knative is a heavy dependency to build images on the cluster. If we're just appending python files, we can do it much simpler and quicker without knative.

Reply:
Knative is the solution that came out after discussing with @jlewi.
You can see some of the context for that in this thread: https://github.com/kubeflow/kubeflow/issues/1240#issuecomment-408878266

Note that @ashahba has a PR opened on the old repo to use configmaps directly and bypass the build step entirely (https://github.com/wbuchwalter/fairing/pull/27). I ll move this PR here once this one get merged. However this approach does not allow for a lot of flexibility since you cannot customize the remote environment.

Curious to hear your idea.